packages/react-native/package.json

```diff
@@ -46,15 +46,17 @@
   "devDependencies": {
     "@formbricks/api": "workspace:*",
     "@formbricks/config-typescript": "workspace:*",
-    "@types/react": "18.3.11",
```

Comment:
I think we should still keep the devDependencies versions there and also add the resolutions